src/parser/parser.c

```diff
@@ -3527,7 +3530,9 @@ static type_t *construct_declarator_type(construct_type_t *construct_list,
 						if (size < 0) {
 							errorf(&size_expression->base.pos, "size of array must be greater than zero");
 						} else if (size == 0 && !GNU_MODE) {
-							errorf(&size_expression->base.pos, "size of array must be greater than zero (zero length arrays are a GCC extension)");
+							warningf(WARN_GNU_EXTENSION, &size_expression->base.pos,
+								"size of array must be greater than zero \
+								(zero length arrays are a GCC extension)");
```

Comment:
You inserted a lot of spaces into the warning message.
Can you put the string back in one line (so you can grep it in the source code)?